app/helpers/application_helper.rb

```diff
@@ -227,7 +227,12 @@ def holding_block_search(document)
         end
         info << content_tag(:div, search_location_display(holding, document), class: 'library-location', data: { location: true, record_id: document['id'], holding_id: id })
       end
-      block << content_tag(:li, info.html_safe, data: { availability_record: check_availability, record_id: document['id'], holding_id: id, aeon: aeon_location?(location), bound_with: document.bound_with? })
+
+      block << if holding['temp_location_code'].present?
+                 content_tag(:li, info.html_safe, class: 'holding-status', data: { availability_record: check_availability, record_id: document['id'], holding_id: id, temp_location_code: holding['temp_location_code'], aeon: aeon_location?(location), bound_with: document.bound_with? })
+               else
+                 content_tag(:li, info.html_safe, class: 'holding-status', data: { availability_record: check_availability, record_id: document['id'], holding_id: id, aeon: aeon_location?(location), bound_with: document.bound_with? })
+               end
```

Comment:
Can we handle the potential for a blank `holding['temp_location_code']` without adding this repetitive conditional?  Either by teaching the js to handle a nil value in data-temp-location-code or something like:

```
block << content_tag(:li, info.html_safe, class: 'holding-status', data: { availability_record: check_availability, record_id: document['id'], holding_id: id, temp_location_code: holding['temp_location_code'], aeon: aeon_location?(location), bound_with: document.bound_with? }.compact)
```

(Adding `.compact` should remove `temp_location_code` from the data hash if it is nil.

Reply:
Done.